.github/bin/script.sh

```diff
@@ -18,6 +18,13 @@ set -e
 
 source ./.github/settings.sh
 
+# Make sure we don't have cc_library rules that use exceptions but do not
```

Comment:
This should reference a github issue which points to these lines to be uncommented when the issue is resolved.

Reply:
Done.